CONTRIBUTING.md

```diff
@@ -1,37 +1,10 @@
-## Develop
+# Contributing Guidelines
 
-### Upgrade the upstream Chart.
+## Upgrading
 
-- First check the upstream changelog and take notes
-
-- Then, find a suitable version.
-
-- Add values diff changelog into `changelog` folder.
-
-- Update `Chart.yaml`:
-  - dependencies
-  - version
-  - appVersion
-
-- Run `helm dependency update`
-
-- Adjust values (If applicable)
-  - Adjust Root Values
-  - Adjust Values in [config repo](https://github.com/giantswarm/config)
-
-- Update docs with `helm-docs`
-
-- Test the Upgrade.
-
-- Release a new app version
-
-- Annonce the change to our customers.
-
-### Update documentation
-
-Chart documentation is generated with [helm-docs](https://github.com/norwoodj/helm-docs) from `values.yaml` file.
-After file modification, regenerate README.md with command:
-
-```bash
-./script/update-docs.sh
-```
+* change the `kube-prometheus-stack` upstream version in Chart dependencies (`helm/prometheus-operator-app/Chart.yaml`)
+* run `helm dependency update helm/prometheus-operator-app` to update the Chart.lock file
```

Comment:
Maybe replace `helm dependency update` with `helm dep up` but that doesn't really change much so I guess it's fine :)

Reply:
I like it to be more explicit :)